systemd/ceph-rbd-mirror.target

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=ceph target allowing to start/stop all ceph-rbd-mirror@.service instances at once
+PartOf=ceph.target
```

Comment:
Only if you have an instance file defined (a la ceph-rbd-mirror@instance1.service)